### PR TITLE
fix(headers): only forward specific headers

### PR DESCRIPTION
**Pull request by beck24**

### CODE REVIEW

**Comment by beck24** [comment]
@mrclay

File: image.php [hunk 1]
```diff
@@ -74,7 +79,28 @@
 	exit;
 }
 
+// we must find a content type of image
+$found_img = false;
+$forwarded_headers = [];
 foreach (explode("\r\n", $headers) as $header) {
+	if (strpos($header, "Content-Type: image/") === 0) {
```
Comment (mrclay) [inline image.php]: Make these comparisons case insensitive.

File: image.php [hunk 2]
```diff
@@ -74,7 +79,28 @@
...
+		$found_img = true;
+		$forwarded_headers[] = $header;
+	}
+	
+	if (strpos($header, "Content-Length: ") === 0) {
```
Comment (mrclay) [inline image.php]: I don't recommend passing this through, but you can send it based on the actual bytes in the image.

**Comment by mrclay** [comment]
PR incoming...

**Pull request closed without merging**